### DISCUSSION TITLE
Brackets text editor not displaying CSS and HTML code

### DISCUSSION THREAD

**Discussion by sumrandom** [brackets-cont/brackets#256] [Q&A]
Hi,

When I try to upload CSS HTML code on brackets nothing shows in the browser view. Am I doing something wrong? This is the code:

<style>
	table {
		border: 1px black solid;
	}

	table th {
		border: 1px black solid;
	}

	table td{
		border: 1px black solid;
	}
</head>
<body>
	<table style="border: 1 px black solid;">
		<tr>
			<th>Player name</th>
			<th>shots</th>
			<th>points</th>
			<th>free throws</th>
			<th>Rebounds</th>
			<th>Steals</th>
			<th>Assists</th>
		</tr>
		<tr>
			<td>jack</td>
			<td>95</td>
			<td>40</td>
			<td>30</td>
			<td>20</td>
			<td>40</td>
			<td>49</td>
		</tr>
		<tr>
			<td>bob</td>
			<td>34</td>
			<td>32</td>
			<td>90</td>
			<td>45</td>
			<td>12</td>
			<td>10</td>
	</table>	
</body>
</html>

**Comment by charlypa** [comment]
Have you imported CSS to html

**Comment by sumrandom** [comment]
How would I do that?
________________________________

**Reply by charlypa** [reply]
Something like this
``` html
<!DOCTYPE html>
<html>
    <head>
        <style>
            table,
            th,
            td {
                border: 1px black solid;
            }
        </style>
    </head>
    <body>
        <table>
            <tr>
                <th>Firstname</th>
                <th>Lastname</th>
            </tr>
            <tr>
                <td>Peter</td>
                <td>Griffin</td>
            </tr>
            <tr>
                <td>Lois</td>
                <td>Griffin</td>
            </tr>
        </table>
    </body>
</html>

```
if you are using a separate style sheet
```html
<link rel="stylesheet" href="mystyle.css">
```
see the full code below
``` html
<!DOCTYPE html>
<html>
    <head>
        <link rel="stylesheet" href="mystyle.css" />
    </head>
    <body>
        <table>
            <tr>
                <th>Firstname</th>
                <th>Lastname</th>
            </tr>
            <tr>
                <td>Peter</td>
                <td>Griffin</td>
            </tr>
            <tr>
                <td>Lois</td>
                <td>Griffin</td>
            </tr>
        </table>
    </body>
</html>

```
please see this for more info
https://www.w3schools.com/css/css_howto.asp
